gremlin/value.go

```diff
@@ -52,14 +53,16 @@ func (v ValueString) String() string {
 const DESC = ValueString("DESC")
 
 // Quote used to quote string values as needed by query
-func Quote(format string, a ...interface{}) ValueString {
-	s := fmt.Sprintf(format, a...)
-	return ValueString(fmt.Sprintf(`"%s"`, s))
+func Quote(s string) ValueString {
+	escaped, err := json.Marshal(s)
+	if err != nil {
+		panic(fmt.Sprintf("Quote failed escaping: %v", err))
```

Comment:
I wouldn't panic here. Even if it is used only for test purpose currently later we could have this used by a user input. I would rather return a string/token that will generate a wrong gremlin expression so that the parser will catch the error. What do you think ?

Reply:
Can `json.Marshal` panic if given a `string` argument? From examining the Go source code I couldn't find such a case, even if the the input string contains illegal UTF8 byte sequences. Since I can't think of such a case, I thought I could completely ignore the `err`, but maybe just a bit clearer with `panic` here. Not sure...

Reply:
@dubek I agree as json.Marshal(<string>) will not generate error ... just ignore the error and add a comment that we ignoring error here because the type is string  and/or return a static ValueString with the error. But don't panic here this could kill other (3rd party) clients

Reply:
Good point, thanks for explaining. I changed this in d3a558d3ff3358f83e2efeaf752fb509b26df874.

What are those 3rd party clients? Might they break with the function signature change I introduced?

```diff
-func Quote(format string, a ...interface{}) ValueString {
+func Quote(s string) ValueString {
```